### ISSUE TITLE
getting BeanDefinitionOverrideException while configuring Sleuth with Jaegar

### ISSUE THREAD

**Issue by sunz1e** [anoophp777/spring-webflux-jaegar-log4j2#1]
Hello Anoop, I am getting following exception while trying to use Open Tracing with Sleuth. 
Did you face the same? 

`The bean 'tracer', defined in class path resource [org/springframework/cloud/sleuth/autoconfig/TraceAutoConfiguration.class], could not be registered. A bean with that name has already been defined in class path resource [io/opentracing/contrib/java/spring/jaeger/starter/JaegerAutoConfiguration.class] and overriding is disabled.\n\nAction:\n\nConsider renaming one of the beans or enabling overriding by setting spring.main.allow-bean-definition-overriding=true\n","logger":"org.springframework.boot.diagnostics.LoggingFailureAnalysisReporter","thread":"main","level":"ERROR"}`

**Comment by anoophp777** [comment]
@meSunnySrivastava no. I've not faced this. Turn on overriding and see if this fixes the issue.

**Comment by sunz1e** [comment]
Hello @anoophp777 , Thanks for replying. 
I was able to resolve the issue. 
I have a question: 
Is it possible to emit Jaeger compatible spans from Sleuth? 
As per my understanding we can let the Zipkin span to be generated and collect it using Zipkin collector from Jaeger correct?
Instead of emitting zipkin Spans can we emit Jaeger span? right now we are dependent upon Zipkin collector.

**Comment by anoophp777** [comment]
Nope. Sleuth doesn't understand Jaeger spans, as far as I know. Things
could have changed now. Check the latest documentation and see. Its been a
while I worked on Jaeger.
